### PR TITLE
A pile of commits. Piling up.

### PR DESCRIPTION
**Pull request by fokep**

### CODE REVIEW

File: _pages/community.md [hunk 1]
```diff
@@ -83,6 +83,30 @@ permalink: /community/
         </a>
         <h3><a href="https://www.reddit.com/r/rvn/" target="_blank">Reddit 2</a></h3>
       </div>
+      <div class="w-1/2 sm:w-1/4 mb-8">
+        <a href="https://www.reddit.com/r/RVNMiner/" target="_blank">
+          <div class="mb-6 py-4 px-5 inline-block rounded-full bg-grey hover:bg-grey-dark">
+            <i class="zmdi zmdi-reddit text-5xl text-white"></i>
+          </div>
+        </a>
+        <h3><a href="https://www.reddit.com/r/RVNMiner/" target="_blank">Reddit 3</a></h3>
```
Comment (mmcshinsky) [inline _pages/community.md]: Reddit (Miners)

**Comment by fokep** [comment]
Add

**Comment by fokep** [comment]
Added another one...

**Comment by fokep** [comment]
Added both asset explorers on the community page

**Pull request merged**